### PR TITLE
Inherit fix for ServusTV  

### PR DESCRIPTION
**Pull request by embeddedc**
See original PR: https://github.com/ytdl-org/youtube-dl/pull/29556

### CODE REVIEW

File: yt_dlp/extractor/servus.py [hunk 1]
```diff
@@ -63,86 +62,51 @@ class ServusIE(InfoExtractor):
...
-        video = self._download_json(
-            'https://sparkle-api.liiift.io/api/v1/stv/channels/international/assets/%s' % video_id,
-            video_id, 'Downloading video JSON', headers={
-                'Authorization': '%s %s' % (token_type, access_token),
-            })
+        if 'GEO_BLOCKED' in attrs.get('playabilityErrors', ''):
+            countries = ', '.join(attrs.get('blockedCountries', ['Unknown']))
+            raise ExtractorError(
+                'Video is geo restricted (restricted countries: %s). '
+                'Try bypassing with --geo-bypass-country option, VPN or --proxy option' % countries, expected=True)
```
Comment (MinePlayersPE) [inline yt_dlp/extractor/servus.py:76]:
```suggestion
        if 'GEO_BLOCKED' in attrs.get('playabilityErrors', []):
            self.raise_geo_restricted(countries=attrs.get('blockedCountries'))
```
Atleast, that's what I think restricted countries means? You might want to also check for the other errors

File: yt_dlp/extractor/servus.py [hunk 2]
```diff
@@ -63,86 +62,51 @@ class ServusIE(InfoExtractor):
...
+        title = attrs.get('title') or video_id
+        description = attrs.get('description')
         series = attrs.get('label')
         season = attrs.get('season')
         episode = attrs.get('chapter')
-        duration = float_or_none(attrs.get('duration'), scale=1000)
-        season_number = int_or_none(self._search_regex(
-            r'Season (\d+)', season or '', 'season number', default=None))
-        episode_number = int_or_none(self._search_regex(
-            r'Episode (\d+)', episode or '', 'episode number', default=None))
```
Comment (MinePlayersPE) [inline yt_dlp/extractor/servus.py:132]: Do you know why these were removed? These regexes seem to still work

Comment (pukkandan) [reply yt_dlp/extractor/servus.py:132]: dont just mark review as resolved without addressing them

**Comment by MinePlayersPE** [comment]
Please commit https://github.com/ytdl-org/youtube-dl/pull/29556/commits/5982cc6ebfca917bc9ce96a0bd8b998e774e0685 as it resolves the current pending review

**Pull request closed without merging**